base/libgit2/repository.jl

```diff
@@ -313,6 +313,24 @@ function checkout_head(repo::GitRepo; options::CheckoutOptions = CheckoutOptions
                  repo.ptr, Ref(options))
 end
 
+"""
+    LibGit2.cherrypick(repo::GitRepo, commit::GitCommit; options::CherrypickOptions = CherrypickOptions())
+
+Cherrypick the commit `commit` and apply the changes in it to the current state of `repo`.
+The keyword argument `options` sets checkout and merge options for the cherrypick. If `null`,
+the default options are used.
+
+!!! note
+    `cherrypick` will *apply* the changes in `commit` but not *commit* them, so `repo` will
+    be left in a dirty state. If you want to apply and commit the changes in `commit` you must
```

Comment:
The first sentence does, but for the second sentence to sound consistent with it I'd say "If you want to also commit the changes"